### ISSUE TITLE
Create service instance dropdown should be grouped

### ISSUE THREAD

**Issue by msecret** [18F/cg-deck#372]
>
If this selection list contains groups of related options, they should be grouped with optgroup.

**Comment by msecret** [comment]
Closing to put in one issue.
